en/docs/developer-guides/workspace-management/overview.md

```diff
@@ -0,0 +1,36 @@
+# Overview
+WSO2 Integrator: BI introduces workspaces – a powerful way to organize and manage complex integration projects through an intuitive graphical interface.
+
+> Note: This feature is available from WSO2 Integrator: BI 1.5.0 onwards and Ballerina version 2201.13.0+.
+
+## What is a Workspace?
```

Comment:
As a practice, we use the camel case only for level 1 headings. For others, we use sentense case.